cql/ManageRareAbnormality.cql

```diff
@@ -752,11 +819,24 @@ define InitialCervicalHistologyInterpretedAsUnspecifiedHsilOrCin2InLast6Years:
       )
   )
 
+define InitialCervicalHistologyInterpretedAsUnspecifiedHsilOrCin2InLast31Years:
+  First(
+    (Collate.SortedBiopsyReports) B
+      where B.date >= Now() - 31 years and
+      B.date occurs after Patient.birthDate.value and
```

Comment:
I'm wondering if this adds anything? If so, why would we check that the test happens after the patient was born here, and not anywhere else?

Reply:
I think this was something I was using during debugging, so this can be removed now.

Reply:
This was removed in the most recent commit. All tests still pass.